### ISSUE TITLE
No importer registered for op: SoftmaxCrossEntropyLoss

### ISSUE THREAD

**Issue by ngockhanh5110** [NVIDIA/TensorRT#2924]
## Description

I am converting model onnx to tensorrt format. 
In the forward method, there is one operation of `nn.CrossEntropyLoss`

## Environment

**TensorRT Version**:  8.5.1
**NVIDIA GPU**: A10
**NVIDIA Driver Version**:  525
**CUDA Version**: 1.8
**CUDNN Version**: 
**Operating System**: Amazon linux 2 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**:   1.13.0
**Baremetal or Container (if so, version)**: 

## Relevant Files

Error log

```text
[05/03/2023-16:06:39] [I] [TRT] No importer registered for op: SoftmaxCrossEntropyLoss. Attempting to import as plugin.
[05/03/2023-16:06:39] [I] [TRT] Searching for plugin: SoftmaxCrossEntropyLoss, plugin_version: 1, plugin_namespace: 
[05/03/2023-16:06:39] [E] [TRT] parsers/onnx/ModelImporter.cpp:726: While parsing node number 6074 [SoftmaxCrossEntropyLoss -> "/loss/SoftmaxCrossEntropyLoss_output_0"]:
[05/03/2023-16:06:39] [E] [TRT] parsers/onnx/ModelImporter.cpp:727: --- Begin node ---
[05/03/2023-16:06:39] [E] [TRT] parsers/onnx/ModelImporter.cpp:728: input: "/Reshape_output_0"
input: "/Reshape_1_output_0"
output: "/loss/SoftmaxCrossEntropyLoss_output_0"
name: "/loss/SoftmaxCrossEntropyLoss"
op_type: "SoftmaxCrossEntropyLoss"
attribute {
  name: "ignore_index"
  i: -100
  type: INT
}
attribute {
  name: "reduction"
  s: "mean"
  type: STRING
}

[05/03/2023-16:06:39] [E] [TRT] parsers/onnx/ModelImporter.cpp:729: --- End node ---
[05/03/2023-16:06:39] [E] [TRT] parsers/onnx/ModelImporter.cpp:731: ERROR: parsers/onnx/builtin_op_importers.cpp:5409 In function importFallbackPluginImporter:
[8] Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"
[05/03/2023-16:06:39] [E] Failed to parse onnx file
[05/03/2023-16:06:39] [I] Finish parsing network model
[05/03/2023-16:06:39] [E] Parsing model failed
[05/03/2023-16:06:39] [E] Failed to create engine from model or file.
[05/03/2023-16:06:39] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec [TensorRT v8501] # trtexec --onnx=/root/working/new/TensorRT/demo/HuggingFace/notebooks/models/gpt2-medium/ONNX/gpt2-medium.onnx --saveEngine=./test.plan --fp16 --explicitBatch --minShapes=input_ids:1x128 --optShapes=input_ids:64x128 --maxShapes=input_ids:128x128 --noDataTransfers --useCudaGraph

```

## Steps To Reproduce
Step 1: Export GPT2 Medium model to ONNX format using torch.onnx.export
```bash
torch.onnx.export(
    model,               # model being run
    inputs,                         # model input (or a tuple for multiple inputs)
    "./gpt2-medium.onnx",   # where to save the model (can be a file or file-like object)
    opset_version=12,          # the ONNX version to export the model to
    input_names=["input_ids"],
    output_names=["loss"],
    dynamic_axes={
        'input_ids' : {0 : 'batch_size', 1: "seq_len"},
        'loss' : {0 : 'batch_size'}
        }
    )
```
Step 2: Run this command
```bash
trtexec --onnx=./gpt2-medium.onnx \
    --saveEngine=./gpt2-medium.plan \
    --fp16 \
    --explicitBatch \
    --minShapes=input_ids:1x128 \
    --optShapes=input_ids:64x128 \
    --maxShapes=input_ids:128x128 \
    --noDataTransfers \
    --useCudaGraph
```

**Comment by zerollzeng** [comment]
I believe an inference model should not contain the loss function, could you please try set `model.eval()` before export?

**Comment by zerollzeng** [comment]
About the error, yes we don't support this operator because TRT is a library target on inference not training.
